module-1/assignments/activity-2.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<tables>
+    <rows>
+        <heading>Empno</heading>
+        <heading>Empname</heading>
+        <heading>Salary</heading>
+    </rows>
```

Comment:
could have used these as tag names for each row instead of creating whole separate element

Reply:
So, It should be like this:-

<rows>
	<Empno>1001</Empno>
      <Empname>Alex</Empname>
      <Salary>3000</Salary>
</rows>
<rows>
      <Empno>1002</Empno>
      <Empname>Sarah</Empname>
      <Salary>4000</Salary>
</rows>
<rows>
	<Empno>1003</Empno>
      <Empname>Peter</Empname>
      <Salary>3500</Salary>
</rows>